acceptance/tests/rototillerTask/command_arguments_with_override.rb

```diff
@@ -0,0 +1,78 @@
+require 'beaker/hosts'
+require 'rakefile_tools'
+
+test_name 'C97827: can set envvar to override command name when using task.command' do
+  extend Beaker::Hosts
+  extend RakefileTools
+
+  step 'Test command with an override_env that has a value' do
+
+    override_env = 'BROCKSAMSON'
+    argument_override_env = 'RUSTYVENTURE'
+    env_key = 'THIS_WAS_IN_ENV'
+    argument_env_key = 'THIS_WAS_THE_ARG_IN_THE_ENV'
+    env_value = 'echo ' << env_key
+
+    default_arg = 'HANKVENTURE'
+
+    @task_name    = 'command_with_args_and_defaults'
+    rakefile_contents = <<-EOS
+$LOAD_PATH.unshift('/root/rototiller/lib')
+require 'rototiller'
+
+Rototiller::Task::RototillerTask.define_task :#{@task_name} do |t|
```

Comment:
we should leave a few tests with this long-form.  but i think most tests should probably use `rototiller_task`, now